docs/docs/Versioning_And_Upgrade/Compatibility.md

```diff
@@ -13,11 +13,5 @@ These restrictions are enforced by the Delphix Engine, and sometimes, the plugin
 
 The Delphix Engine will enforce these rules before a newly-uploded plugin is allowed to be installed:
 
-* The [build number](/Versioning_And_Upgrading/Versioning/#build-number) may only move forward, not backwards.
-* All [data migration IDs](/References/Glossary/#data-migration-id) that are present in the already-installed plugin must also be present on the newly-uploaded plugin. (The newly-uploaded plugin may add more data migrations, of course.)
-
-<!--
-## Plugin-Defined Compatibility
-
-TODO: How can the plugin influence which version changes are allowed/denied?
--->
```

Comment:
Does it make sense to leave this comment right below just so we have an idea of where we do want to add this information once we implement it? Fine with removing it but I think the TODO is a good reminder if we actually think we're gonna do the plugin defined stuff eventually.

Reply:
I'd rather remove it, we should add it back when its there. These sections might move around and the TODO might not be correct when we do actually implement compatibility checks, it could get messy to always move the TODO to the correct place. I think temporary TODOs are ok for things that we are adding very very soon, but otherwise they can quickly get out of date.